pages/conference/keynotes.vue

```diff
@@ -137,7 +137,8 @@ export default {
     },
     methods: {
         getKeynoteId(keynote) {
-            return keynote.speaker.name_en_us.split(' ').join('_')
+            // adjust for schedule data format
+            return keynote.speaker.name_zh_hant.split(' ').join('_')
```

Comment:
why does this change needed?

Reply:
I found that the scroll behavior on production page is failed because the anchor and the url are inconsistent.
Reproduce steps:
1. go to PyCon Taiwan 2021 official site
2. head to schedule page
3. click on the first keynote by 魏澤人
4. being directed to keynote page without scroll behavior

Thus, modifying the data used as id can fix the issue.

Reply:
I always get the english content on [schedule page](https://tw.pycon.org/2021/zh-hant/conference/schedule) (cannot change with locale switch) and thus the generated id is correct for me. I assume you get the zh-hant content on this page?

![image](https://user-images.githubusercontent.com/24987826/134337056-8a1812a7-b74d-4d21-b743-ec9d8919b8fd.png)

Reply:
Yes ~ I thought that's why the scroll behavior failed.

<img width="1872" alt="截圖 2021-09-26 下午12 05 10" src="https://user-images.githubusercontent.com/13831865/134793232-57680ac1-a952-47ff-b48d-cf4c7835fbea.png">